dask_glm/algorithms.py

```diff
@@ -11,7 +11,7 @@
 from scipy.optimize import fmin_l_bfgs_b
 
 
-from dask_glm.utils import dot, normalize, scatter_array, get_distributed_client
+from dask_glm.utils import dot, normalize, scatter_array, get_distributed_client, safe_zeros_like
```

Comment:
Where is `safe_zeros_like` coming from? I suppose you wanted to `from dask.array.utils import zeros_like_safe` instead, from https://github.com/dask/dask/blob/48a4d4a5c5769f6b78881adeb1b3973a950e5f43/dask/array/utils.py#L350